linux-4.4.defconfig

```diff
@@ -78,7 +78,6 @@ CONFIG_SYN_COOKIES=y
 # CONFIG_IPV6 is not set
 CONFIG_CFG80211=y
 CONFIG_CFG80211_INTERNAL_REGDB=y
-CONFIG_CFG80211_WEXT=y
```

Comment:
It turns out that the RTL8192CU wifi driver requires wext so this option is pulled in automatically. In other words, it's no longer needed in the defconfig. See:
https://github.com/raspberrypi/linux/blob/rpi-4.4.y/drivers/net/wireless/realtek/rtl8192cu/Kconfig#L4